docs/en/operations/access_rights.md

```diff
@@ -104,5 +104,10 @@ The user can get a list of all databases and tables in them by using `SHOW` quer
 
 Database access is not related to the [readonly](settings/permissions_for_queries.md#settings_readonly) setting. You can't grant full access to one database and `readonly` access to another one.
 
+## ZooKeeper Authenticaton {#zookeeper-authentification}
```

Comment:
I'd move this to the article about ZooKeeper configuration, this one is about access rights inside ClickHouse itself